phy/io/kwik/store_items.py

```diff
@@ -528,8 +528,8 @@ def n_unmasked_channels(self, cluster):
 
     def mean_probe_position(self, cluster):
         mean_masks = self.load(cluster, 'mean_masks')
-        return _mean(self.model.probe.positions *
-                     mean_masks[:, np.newaxis], (2,))
+        return np.sum(self.model.probe.positions[:, 1] *
+                      mean_masks) / np.sum(mean_masks)
```

Comment:
* this should return the mean position with two coordinates x and y, not just y
* you should create a new `_sum()` function, like the `_mean` one, to deal with empty arrays